testplan/web_ui/testing/src/Report/InteractiveReport.js

```diff
@@ -72,6 +74,7 @@ class InteractiveReportComponent extends BaseReport {
       navWidth: `${INTERACTIVE_COL_WIDTH}em`,
       resetting: false,
       reloading: false,
+      running: false,
```

Comment:
So I looked at this new status and wonder two things:
- Can we do without it?
- If not, then we need to block certain actions when it is True. Right now I think we can trigger it twice without problem, send a reload during it, reset, etc... See for instance, how the reloadCode in InteractiveReport is handling it.

Reply:
> Can we do without it?

We only need that for logging the state of the progress. However, we could use another variable eg.: _runtime_status_ which could replace _resetting_, _reloading_, _aborting_ as well, but in that case need to refactor the related functionalities.

> If not, then we need to block certain actions when it is True. Right now I think we can trigger it twice without problem, send a reload during it, reset, etc... See for instance, how the reloadCode in InteractiveReport is handling it.

That makes sense. I will update the code.